### ISSUE TITLE
BOM update

### ISSUE THREAD

**Issue by Edison517** [jamesturton/bose-qc35-usb-c#8]
I may have missed it, but I had to DNP R1-5 to get the pcb to work when installed...I only tried that because of the picture you posted of the assembled board...I also was testing the LED orientation and all 4 would light up with R1-5 populated, but then they test correctly, individually when R1-5 are DNP. Just my $0.02. Great work, otherwise! Thanks!

**Comment by jamesturton** [comment]
Did you use `CG0402MLC-05LG` for R1-4?
I didn't mount anything in those as you saw :wink: So I've updated the schematic to make this much clearer!

**Comment by Edison517** [comment]
no, I didn't take a look at the schematic at all (I don't have KiCad installed), so I didn't see until today that I was supposed to use those varistors and not 0ohm resistors (oops!). I'd like to see a schematic pdf posted ;-) and I ordered those varistors to install later.

[update]
I got the varistors, placed them, but then my blue led wouldn't turn off even when it was OFF. I removed R1-4, 17-19, and it works great now! I hope I didn't fry anything with my original part placement, but it seems to work now.

Thanks for designing a really cool mod/upgrade for these headphones!
